colossus-metrics/src/main/scala/colossus/metrics/collectors/Rate.scala

```diff
@@ -4,115 +4,19 @@ import akka.actor._
 
 import scala.concurrent.duration._
 
-import EventLocality._
+class Rate(val address: MetricAddress)(implicit collection: Collection) extends Collector {
```

Comment:
This class shares a lot in common with Counter.  Maybe extract a Trait or something?  I know in general, they are closely related metric types, so it kind of makes sense that their implementations are so identical